perf/istio-install/create_cluster.sh

```diff
@@ -51,40 +53,39 @@ MAXPODS_PER_NODE=100
 # Labels and version
 ISTIO_VERSION=${ISTIO_VERSION:?"Istio version label"}
 DEFAULT_GKE_VERSION=$(default_cluster "${ZONE}")
+# shellcheck disable=SC2181
 if [[ $? -ne 0 ]];then
   echo "${DEFAULT_GKE_VERSION}"
   exit 1
 fi
 GKE_VERSION=${GKE_VERSION-${DEFAULT_GKE_VERSION}}
 
-# default scope for reference
-SCOPES_DEFAULT="https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append"
```

Comment:
Please keep the default scope in comments.

Reply:
It should continue to be $*

“$*” is incorrect .
Looks like shell linter does not understand intent.